docs/current_docs/user-guide/ci/snippets/azure-pipelines.yml

```diff
@@ -9,5 +9,5 @@ steps:
 - script: curl -L https://dl.dagger.io/dagger/install.sh | BIN_DIR=$HOME/.local/bin sh
   displayName: 'Install Dagger CLI'
 
-- script: dagger call -m github.com/shykes/daggerverse/hello@v0.1.2 hello --greeting Hola --name Daggernauts --shout
+- script: dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.5 call build --project=. --args=./cmd/dagger
```

Comment:
@kpenfound where should the cloud token go in this config?

Reply:
below script you'd have
`env: DAGGER_CLOUD_TOKEN: $(dagger_cloud_token)`

where you've set the variable
```yaml
variables:
- name: dagger_cloud_token
   value: <TOKEN HERE>
```

https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#understand-variable-syntax

https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#systemaccesstoken


Although this requires committing the token, which we aren't recommending anymore. The docs have some guides on integrating with Azure Key Vault, but I don't have any first hand experience with that.

https://learn.microsoft.com/en-us/azure/devops/pipelines/release/azure-key-vault?view=azure-devops&tabs=yaml#create-a-new-pipeline

Reply:
@kpenfound from my reading, I don't think it's necessary to commit the token or have a separate `variables` block. In the first link you sent, it shows:

```
variables:
 GLOBAL_MYSECRET: $(mySecret) # this will not work because the secret variable needs to be mapped as env
 ...
- bash: |
...
  env:
    MY_MAPPED_ENV_VAR: $(mySecret) # the recommended way to map to an env variable
```

and in the explanation it says:

```
You need to set secret variables in the pipeline settings UI for your pipeline...You need to explicitly map secret variables.
```

I've revised the example now omitting the `variables` block. Let me know if you agree or if we should change it further?

Reply:
I missed the secret variables section! Yes that looks good to me